### DISCUSSION TITLE
Developer docs (how to create custom modules)

### DISCUSSION THREAD

**Discussion by andreadev-it** [nvim-neorg/neorg#776] [Ideas]
Hi everyone,
I just wanted to share the first part of a tutorial series about how to create custom neorg modules. Since all the informations I got on how to build plugins are reverse-engineered, if anyone has any experience with neorg modules, please watch it and let me know if there are problems with it. I did it just to help sharing neorg and kickstart new modules for it (I'm not monetizing my channel, also, it has only one video right now).

This is a link to the [youtube video](https://youtu.be/j4lTvIGRhmw)
This is a [markdown version](https://github.com/andreadev-it/neorg-module-tutorials/blob/main/introduction.md) that, if anyone on the team wants, can be included in the wiki

Hope this might be helpful, have a nice day 🙂

**Comment by vhyrro** [comment]
This is super awesome! Thanks for taking the time to do this, the video was great :)

I added a link to the markdown version in the `Home.md` page for the wiki, once the videos get a few more episodes I'll link those too!

**Reply by andreadev-it** [reply]
I'm happy to see you liked it 🙂 
I'll post here when new tutorials will be available, hopefully soon 👍🏼

**Reply by andreadev-it** [reply]
Hi,
As of now, parts 2 and 3 are out both in video and markdown form.
If anyone could double check the info so that we're sure that they are qualitative, it would be very helpful 🙂

Part 2: https://www.youtube.com/watch?v=F64TD4nl7J8
Part 3: https://www.youtube.com/watch?v=E1Wp4YMEmyA
